docs/en/02_Developer_Guides/08_Performance/02_HTTP_Cache_Headers.md

```diff
@@ -31,3 +31,23 @@ clause in `Cache-Control` and `Pragma` will be included.
 
 Used to set the modification date to something more recent than the default. [api:DataObject::__construct] calls 
 [api:HTTP::register_modification_date(] whenever a record comes from the database ensuring the newest date is present.
+
+### Vary: cache header
+
+By default, SilverStripe will output a `Vary` header (used by upstream caches for determining uniqueness) 
+that looks like
+
+```
+Cookie, X-Forwarded-Protocol, User-Agent, Accept
+```
+
+To change the value of the `Vary` header, you can change this value by specifying the header in configuration
+
+```yml
+HTTP:
+  vary: ""
```

Comment:
Because it's defined as a string, and the alternate is provided as an explicit empty string, it does - if it was an array of options, it'd just merge the existing array together with the new array and not override anything.

At least, it works that way if calling Config::update() and passing in an empty string in the testcase, and by manual testing with a config.yml in mysite/_config/. If it's not _meant_ to, well - config voodoo :eyes: :)